### PR TITLE
Add GQLstatus to `DynamicProperty` and `CodeGenerationFailed`

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-149.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-149.surge.sh/status-codes/5/notifications/all-notifications)

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 1]
```diff
@@ -509,8 +618,25 @@ m|INFORMATION
...
+======
+[.include-with-GQLSTATUS-code]
+======
+
+[cols="<1s,<4"]
+|===
+|GQLSTATUS code
+m|01N40
+|StatusDescription
+a|info: runtime unsupported.
```
Comment (Lojjs) [inline modules/ROOT/pages/notifications/all-notifications.adoc:630]:
```suggestion
a|warn: runtime unsupported.
```

File: modules/ROOT/pages/notifications/all-notifications.adoc [hunk 2]
```diff
@@ -509,8 +618,25 @@ m|INFORMATION
...
+The query cannot be executed `runtime=pipelined operatorEngine=compiled expressionEngine=compiled`, `runtime=pipelined operatorEngine=interpreted expressionEngine=compiled` is used. Cause: `method too big`.
```
Comment (Lojjs) [inline modules/ROOT/pages/notifications/all-notifications.adoc:631]:
```suggestion
The query cannot be executed with `preparser_input1`, `preparser_input2` is used. Cause: `$msg`.
```

This is the general status description and not an example (the reason there is no example is this warning is kind of internal and only occur for things like memory errors in runtime so it is a bit hard to make a good example)

**Review by Lojjs — CHANGES_REQUESTED** [review]
A few modifications needed. I also realised something is a bit strange on our side with 01N40, wrote a Slack message about it. If we need to change something about it, that can be done separately from this PR

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-149.surge.sh

Updated pages:
- [List of all notification codes](https://neo4j-docs-status-codes-149.surge.sh/status-codes/5/notifications/all-notifications)

**Review by Lojjs — APPROVED** [review]

**Pull request merged**